static/app/components/smartSearchBar/index.tsx

```diff
@@ -337,8 +251,22 @@ class SmartSearchBar extends React.Component<Props, State> {
     tags: {},
     dropdownVisible: false,
     loading: false,
+    actionsVisible: this.props.actionBarItems?.length ?? 0,
```

Comment:
I would rename this to something that reflects that this is a number (vs `dropdownVisible` which is similar but a boolean).

Reply:
👍